kpops/components/base_components/models/to_section.py

```diff
@@ -28,6 +28,7 @@ class TopicConfig(BaseModel):
 
     class Config(BaseConfig):
         extra = Extra.forbid
+        allow_population_by_field_name = True
```

Comment:
Does this mean that we use the field name (like `shema_registry_url`) in the `pipeline.yaml`?

Reply:
yes, both snake case and camel case version are supported, although snake case should be preferred